### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Hzaatiti**
Added new content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
 Content goes here.
+  This edit comes from a new pull request.
```
Comment (hzaatiti-NYU) [inline index.html]: Remove the beginning space

File: index.html [hunk 2]
```diff
@@ -7,5 +7,6 @@
...
 </body>
 </html>
```
Comment (hzaatiti-NYU) [inline index.html:14]: add an image here

**Review by hzaatiti-NYU — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Hzaatiti) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (Hzaatiti) [reply index.html:14]: done

File: index.html [hunk 3]
```diff
@@ -7,5 +7,7 @@
 </head>
 <body>
 Content goes here.
+This edit comes from a new pull request.
```
Comment (hzaatiti-NYU) [inline index.html:10]: This looks great!

**Review by hzaatiti-NYU — APPROVED** [review]
All good

**Comment by hzaatiti-NYU** [comment]
#3 issue raises a problem with this already merged PR

**Pull request merged**